packages/rainbowkit/src/wallets/connectorsForWallets.ts

```diff
@@ -1,36 +1,28 @@
-import { Connector } from 'wagmi';
-import { WalletConnectConnector } from 'wagmi/connectors/walletConnect';
+import type { CreateConnectorFn } from 'wagmi';
 import { isHexString } from '../utils/colors';
-import { isMobile } from '../utils/isMobile';
 import { omitUndefinedValues } from '../utils/omitUndefinedValues';
-import { Wallet, WalletInstance, WalletList } from './Wallet';
+import type { Wallet, WalletDetailsParams, WalletList } from './Wallet';
 
 interface WalletListItem extends Wallet {
   index: number;
   groupIndex: number;
   groupName: string;
 }
 
-/*
-  Build your own list of wallets for the RainbowKit modal
-  with their necessary connectors. This way you have full
-  control over which wallets to display, and in which order.
-  Returns a `connectors` fn to pass directly to Wagmi's `createConfig`. 
-*/
-export function connectorsForWallets(walletList: WalletList): () => Connector[];
-
 /*
   Assemble a list of low-level connectors for use with the
   `WalletButton` and `RainbowButton` components in custom implementations.
 */
-export function connectorsForWallets(wallets: Wallet[]): Connector[];
+export function connectorsForWallets(
+  wallets: Wallet[] | WalletList,
+): CreateConnectorFn[];
```

Comment:
I think we should accept `appName`, `appDescription`, and `appUrl` here so that it's passed to the WalletConnect connectors through this flow too